client/my-sites/term-tree-selector/docs/example.jsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PureRenderMixin from 'react-pure-render/mixin';
+
+/**
+ * Internal dependencies
+ */
+import TermTreeSelector from '../';
+import observe from 'lib/mixins/data-observe';
+import sites from 'lib/sites-list';
+
+const TermTreeSelectorExample = React.createClass( {
+	mixins: [ observe( 'sites' ), PureRenderMixin ],
+
+	render() {
+		const primary = this.props.sites.getPrimary();
+
+		/* eslint-disable wpcalypso/jsx-classname-namespace */
```

Comment:
Hmm, seems we need a better solution here moving forward. Maybe a change on the directory name for DevDocs examples to, e.g. `docs-example` (in addition or in place of `design-assets__group`). Or perhaps move all examples to a single folder `design-assets` somewhere.

/cc @mtias